test/Test_CarbonVote.js

```diff
@@ -1,46 +1,61 @@
+// accounts setting in this test environment:
```

Comment:
rename filename to carbonVote.js.   testCarbonVote.js is also acceptable but Test_CarbonVote.js is not good